packages/salesforcedx-vscode-core/package.json

```diff
@@ -104,6 +104,10 @@
         {
           "command": "sfdx.force.lightning.interface.create",
           "when": "explorerResourceIsFolder && sfdx:project_opened"
+        },
+        {
+          "command": "sfdx.force.debugger.stop",
```

Comment:
The debugger stop command was purposely removed from explorer menu in https://github.com/forcedotcom/salesforcedx-vscode/pull/144.